### PR TITLE
EAR-1469  disabled toggle if no summary

### PR DESCRIPTION
**Pull request by jasehumphr**
> ### BEFORE MAKING YOUR PR
>
> Please ensure:
>
> - There are no linting errors, all tests must pass
> - PR is named after JIRA ticket number e.g. EAR-###
> - **Accesibility** checks are completed:
>   - Elements have discernible and consistent focus states
>   - Elements can be navigated to and used by just a keyboard
>   - Elements and text can be read out by a screen reader, and the descriptions are understandable
> - Your feature / bug fix works across **GCP** and **AWS** (where appropriate)
>   - Are modifications to eq-publisher-v3 required?
>   - Are modifications to the Firestore data source required?

---

### What is the context of this PR?

> Describe what you have changed as part of this task and why; include a link to the Trello card/GitHub issue, related pull requests, etc as required.
>
> E.g. _"Currently, if you add a new question to Author the entire app crashed. This is because..."_

### How to review

> Add to the list below as appropriate, including screenshots when necessary

- Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
  - ensure it can be opened in Author;
  - then, ensure it can be viewed in Runner by pressing the **view survey** button
- Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process

### CODE REVIEW

File: eq-author/src/App/section/Design/SectionEditor/SectionSummary/index.js [hunk 1]
```diff
@@ -64,26 +69,28 @@ const SectionSummary = ({ id, sectionSummary, collapsibleSummary }) => {
...
-            checked={collapsibleSummary}
-          />
-        </ToggleWrapper>
-      </InlineField>
+      <EnableDisableWrapper disabled={!sectionSummary}>
+        <InlineField>
+          <Label htmlFor="required-completed">Collapsible summary</Label>
+          <ToggleWrapper>
+            <ToggleSwitch
+              disabled="true"
```
Comment (MattElcock) [inline eq-author/src/App/section/Design/SectionEditor/SectionSummary/index.js:77]: Should this always be set to disabled?

**Review by MattElcock — APPROVED** [review]

**Review by MattElcock — CHANGES_REQUESTED** [review]
You haven't written any tests.

File: eq-author/src/App/section/Design/SectionEditor/SectionSummary/index.js (hunk 1 above)
Comment (MattElcock) [inline eq-author/src/App/section/Design/SectionEditor/SectionSummary/index.js:77]: Should this always be disabled?

**Review by MattElcock — CHANGES_REQUESTED** [review]

**Review by ashtuck — APPROVED** [review]

File: eq-author/src/App/section/Design/SectionEditor/HubSettings/index.test.js [hunk 2]
```diff
@@ -13,11 +18,13 @@ describe("Section HubSettings", () => {
...
-      props = {
-        id: "testID1",
-        requiredCompleted: false,
-        showOnHub: false,
-      }
+    props = {
+      id: "testID1",
+      requiredCompleted: false,
+      showOnHub: false,
+      sectionSummary: false,
```
Comment (martyncolmer) [inline eq-author/src/App/section/Design/SectionEditor/HubSettings/index.test.js:25]: Didn't need to add this and collapsible summery here.

**Review by martyncolmer — CHANGES_REQUESTED** [review]

File: eq-author/src/App/section/Design/SectionEditor/HubSettings/index.test.js (hunk 2 above)
Comment (martyncolmer) [reply eq-author/src/App/section/Design/SectionEditor/HubSettings/index.test.js:25]: You can remove sectionSummary and collapsible summary, their redundant,

**Review by farres1 — COMMENTED** [review]
The feature looks really good! There was one bug I noticed.

1 - Enable section summary
2 - Enable collapsible summary
3 - Disable section summary

The collapsible summary toggle cannot be clicked, but the collapsible summary toggle remains enabled (its value is `true` in the JSON data)

**Review by farres1 — APPROVED** [review]

**Review by ashtuck — APPROVED** [review]

**Pull request merged**